docs/units/format.rst

```diff
@@ -1,27 +1,33 @@
 .. _astropy-units-format:
 
-String representations of units
+String Representations of Units
 *******************************
 
-.. |quantity| replace:: :class:`~astropy.units.Quantity`
+.. |Quantity| replace:: :class:`~astropy.units.Quantity`
 
-.. |unit| replace:: :class:`~astropy.units.UnitBase`
+.. |Unit| replace:: :class:`~astropy.units.UnitBase`
 
-Converting units to string representations
+Converting Units to String Representations
 ==========================================
 
-You can control the way that |quantity| and |unit| objects are rendered as
+You can control the way that |Quantity| and |Unit| objects are rendered as
 strings using the new `Format String Syntax
-<https://docs.python.org/3/library/string.html#format-string-syntax>`__.
-New-style format strings use the ``"{}".format()`` syntax.  Most of
-the format specifiers are similar to the old ``%``-style formatting,
-so things like ``0.003f`` still work, just in the form
-``"{:0.003f}".format()``.
+<https://docs.python.org/3/library/string.html#format-string-syntax>`_.
+New-style format strings use the ``"{}".format()`` syntax. Most of the format
+specifiers are similar to the old ``%``-style formatting, so things like
+``0.003f`` still work, but in the form ``"{:0.003f}".format()``.
 
 For quantities, format specifiers, like ``0.003f`` will be applied to
-the |quantity| value, without affecting the unit. Specifiers like
+the |Quantity| value, without affecting the unit. Specifiers like
 ``20s``, which would only apply to a string, will be applied to the
-whole string representation of the |quantity|. This means you can do::
+whole string representation of the |Quantity|.
+
+Examples
+--------
+
+.. EXAMPLE START: Converting Units to String Representations
+
+To render |Quantity| or |Unit| objects as strings::
 
     >>> from astropy import units as u
     >>> import numpy as np
```

Comment:
I guess a question is if we should change the example (or at least some of the examples) to f-strings...

Reply:
Good idea, @adrn will add some commits to this with updates to the code.

Reply:
The f-string examples do look nicer, thanks both!